service-api/module/Application/src/Yoti/YotiService.php

```diff
@@ -265,6 +265,39 @@ public function retrieveLetterPDF(string $sessionId, string $nonce, int $timesta
         return ["status" => "PDF Created", "pdfBase64" => $base64];
     }
 
+    public function retrieveMedia(string $sessionId, string $mediaId, string $nonce, int $timestamp): array
+    {
+        $headers = $this->getSignedRequest(
+            '/sessions/' . $sessionId . '/media/' . $mediaId . '/content',
+            'GET',
+            $nonce,
+            $timestamp,
+            null,
+            $sessionId,
+            $mediaId
+        );
+        try {
+            $document = $this->client->get('/idverify/v1/sessions/' . $sessionId . '/media/' . $mediaId . '/content', [
+                'headers' => $headers,
+                'query' => [
+                    'sdkId' => $this->sdkId->getValue(),
+                    'sessionId' => $sessionId,
+                    'mediaId' => $mediaId,
+                    'nonce' => $nonce,
+                    'timestamp' => $timestamp
+                ],
+            ]);
+        } catch (GuzzleException $e) {
+            $this->logger->error('Unable to connect to Yoti service [' . $e->getMessage() . '] ', [
+                'data' => [ 'operation' => 'retrieve-media', 'header' => $headers]
+            ]);
+            throw new YotiException("A connection error occurred. Previous: " . $e->getMessage());
+        }
+        $response = json_decode(strval($document->getBody()), true);
+
+        return ["status" => "Media retrieved", "response" => $response];
```

Comment:
This string isn't used anywhere, so I don't see any need to add it:
```suggestion
        return ["response" => $response];
```